data/catalogue.csv

```diff
@@ -179,16 +179,17 @@ dateAdded,dateUpdated,url,name,description,documentation,license,maintainer,area
 2020-10-12,2020-10-28,https://data.police.uk/api/,Police API,"The API provides a rich data source for information, including:  Neighbourhood team members Upcoming events Street-level crime and outcome data Nearest police stations The API is implemented as a standard JSON web service using HTTP GET and POST requests. Full request and response examples are provided in the documentation.",https://data.police.uk/docs/,,https://data.police.uk/contact/,,,,uk-police
 2020-10-12,2020-10-28,https://api.tfl.gov.uk/,Transport for London Unified API,"To use the Unified API, developers should register for an Application ID and Key. Append the app_id and app_key query parameters to your requests.  The public TfL data (or 'open data') released here is for open data users to use in their own software and services. We encourage software developers to use this data to present customer travel information in innovative ways - providing they adhere to the transport data terms and conditions.",https://api.tfl.gov.uk/swagger/docs/v1,,,,,,transport-for-london
 2020-10-12,2020-10-28,https://github.com/DFE-Digital/eapim-platform-api,DfE EAPIM Platform API,"Enterprise API Management Platform API for managing applications, subscriptions & retreiving api details",https://github.com/DFE-Digital/eapim-platform-api,,,,,,department-for-education
-2020-11-05,2020-11-05,https://epc.opendatacommunities.org/api/v1/domestic/search,Energy Performance Certificates - Domestic Buildings Search API,Allows searching and filtering of our property-level open data on Energy Performance of buildings by one or any combination of the following attribules.: Address;  Postcode; Local Authority; Parliamentary Consistency; type of property; Energy band; date that the EPC was lodged on the register,https://epc.opendatacommunities.org/docs/api/domestic#domestic-search,https://epc.opendatacommunities.org/docs/copyright,odc@comunities.gov.uk,England and Wales,2018-02-26,optional,ministry-for-housing-communities-and-local-government
+2020-11-05,2020-11-05,https://epc.opendatacommunities.org/api/v1/domestic/search,Energy Performance Certificates - Domestic Buildings Search API,Allows searching and filtering of our property-level open data on Energy Performance of buildings by one or any combination of the following attribules.: Address,  Postcode, Local Authority, Parliamentary Consistency, type of property, Energy band, date that the EPC was lodged on the register,https://epc.opendatacommunities.org/docs/api/domestic#domestic-search
```

Comment:
This record doesn't look right.

```
any combination of the following attribules.: Address,  Postcode, Local Authority,
```

Besides "attribules" which should be "attributes", is the comma-separated list a set of fields? if not, it needs wrapping in double quotes.